src/main/resources/liquibase/042-updateDB.xml

```diff
@@ -12,13 +12,74 @@
     </changeSet>
     <changeSet author="simone" id="setDeletedFlag">
         <sql>UPDATE ddp_medical_record as m, (
-                SELECT m.medical_record_id from ddp_medical_record m
-                left join ddp_institution i on (i.institution_id = m.institution_id)
-                left join ddp_participant p on (p.participant_id = i.participant_id)
-                left join ddp_instance realm on (p.ddp_instance_id = realm.ddp_instance_id)
-                where realm.instance_name = 'Prostate' and (p.ddp_participant_id like '1771332079%' OR p.ddp_participant_id like '-763171247%') and type = 'INITIAL_BIOPSY' and i.ddp_institution_id != '1' and i.institution_id &#60; 20000) as m2
-            SET m.deleted = 1 WHERE m2.medical_record_id = m.medical_record_id;</sql>
+            SELECT m.medical_record_id from ddp_medical_record m
+            left join ddp_institution i on (i.institution_id = m.institution_id)
+            left join ddp_participant p on (p.participant_id = i.participant_id)
+            left join ddp_instance realm on (p.ddp_instance_id = realm.ddp_instance_id)
+            where realm.instance_name = 'Prostate' and (p.ddp_participant_id like '1771332079%' OR p.ddp_participant_id
+            like '-763171247%') and type = 'INITIAL_BIOPSY' and i.ddp_institution_id != '1' and i.institution_id &#60;
+            20000) as m2
+            SET m.deleted = 1 WHERE m2.medical_record_id = m.medical_record_id;
+        </sql>
     </changeSet>
     <!-- prefix for gen2 prostate pts in dev: 1771332079 on prod: -763171247  -->
     <!-- prod i.instance_id > 20000 are actual new added initial_biopsies for migrated pts  -->
-</databaseChangeLog>
\ No newline at end of file
+
+    <changeSet id="carrier_per_study_key" author="pegah">
+        <addColumn tableName="carrier_service">
+            <column name="carrier_tracking_url" type="VARCHAR(500)"></column>
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="DDP-5445_ups_shipment_table" author="pegah">
+        <createTable tableName="ups_shipment">
+            <column name="ups_shipment_id" type="INT" autoIncrement="true">
+                <constraints nullable="false" primaryKey="true"></constraints>
+            </column>
+            <column name="dsm_kit_request_id" type="INT">
+                <constraints nullable="false"></constraints>
+            </column>
+        </createTable>
+
+    </changeSet>
+    <changeSet id="DDP-5445-ups_package-table" author="pegah">
```

Comment:
Are there logical unique keys that can be added to each table to prevent application bugs such as repeatedly adding accidentally duplicated rows?